metadata/src/main/java/org/apache/kafka/controller/QuorumController.java

```diff
@@ -302,12 +302,30 @@ public QuorumController build() throws Exception {
             KafkaEventQueue queue = null;
             try {
                 queue = new KafkaEventQueue(time, logContext, threadNamePrefix + "QuorumController");
-                return new QuorumController(logContext, nodeId, clusterId, queue, time,
-                    configSchema, raftClient, quorumFeatures, defaultReplicationFactor,
-                    defaultNumPartitions, isLeaderRecoverySupported, replicaPlacer, snapshotMaxNewRecordBytes,
-                    leaderImbalanceCheckIntervalNs, sessionTimeoutNs, controllerMetrics,
-                    createTopicPolicy, alterConfigPolicy, configurationValidator, authorizer,
-                    staticConfig, bootstrapMetadata);
+                return new QuorumController(
```

Comment:
sigh. all right, fine. I guess we might as well do one line per parameter.

Reply:
Hehe. I thought it was improvement. I always find the previous style harder to read and update.